src/main/java/br/com/metricminer2/scm/GitRemoteRepository.java

```diff
@@ -0,0 +1,182 @@
+package br.com.metricminer2.scm;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.InvalidRemoteException;
+import org.eclipse.jgit.api.errors.TransportException;
+
+import br.com.metricminer2.domain.ChangeSet;
+import br.com.metricminer2.domain.Commit;
+
+public class GitRemoteRepository implements SCM {
+	
+	private GitRepository tempGitRepository;
+	

```

Comment:
just as a matter of style: remove these enters here, so that all attributes are together with each other. That's how it is in all other classes.